corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -1467,6 +1467,20 @@ def get_ledger_transactions_for_form(form_id, limit_to_cases):
             for trans in resultset:
                 yield trans
 
+    @staticmethod
+    def get_section_and_entry_combinations(domain):
+        section_entry_combos = set()
+        for db_name in get_db_aliases_for_partitioned_query():
```

Comment:
Because this is only used in one place, it felt like a bit overkill to create a new proxy function for this. Later I realized, I could have probably implemented these queries in `corehq/apps/exports` instead of in the accessors, but neither one seems particularly better than the other. Happy to change this if its desired